copydb/copydb.go

```diff
@@ -112,6 +113,7 @@ func (this *CopydbFerry) Run() {
 
 	this.Ferry.StopTargetVerifier()
 
+	this.Ferry.EndCutover(cutoverStart)
```

Comment:
Similar, I would add a comment. This flow is already a bit confusing, adding more without a comment will likely add to confusion.